mods/ra/chrome.yaml

```diff
@@ -575,7 +575,7 @@ button-disabled: dialog.png
 	corner-tr: 639,256,1,1
 	corner-bl: 512,383,1,1
 	corner-br: 639,383,1,1  
-    
+	
```

Comment:
Instead of adding a tab here, could you just empty the line completely, please?
(Same below.)